curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use--to-import-everything-from-a-file.english.md

```diff
@@ -16,7 +16,7 @@ You may use any name following the <code>import * as </code>portion of the state
 
 ## Instructions
 <section id='instructions'>
-The code in this file requires the contents of another file, <code>"capitalize_strings"</code>, that is in the same directory as the current file. Add the appropriate <code>import *</code> statement to the top of the file, using the object provided.
+The code in this file requires the contents of another file, <code>"capitalize_strings"</code>, that is in the same directory as the current file. Add the appropriate <code>import * as</code> statement to the top of the file. You can name the object that receives the imported values whatever you'd like.
```

Comment:
Is this necessary?  The paragraph above already has
> You may use any name following the `<code>import * as </code>`portion of the statement

Adding the 'as' is definitely useful, though. Could you add it at the start of the description, line 9, too?

Reply:
It is a bit redundant. Removed that last sentence and added "as" to line 9.

Reply:
That looks a lot better without 'using the object provided'.  I suspect that line was relevant at some point, but now it's just confusing.

Thanks for sorting that out.

Reply:
Thanks :)